frontend/src/components/opportunity/OpportunityDescription.tsx

```diff
@@ -127,10 +131,39 @@ const OpportunityDescription = ({ summary }: Props) => {
     "--"
   );
 
+  const downloadNOFO = () => {
+    window.open(nofoPath, "_blank")?.focus();
```

Comment:
My understanding is that in some browsers they handle window open differently and for best cross browser support of downloading files one should add the focus, but to be fair not an expert, so open to more research/input if need be.

Reply:
Upon further review, removing the focus because I think it adds unnecessary code complexity/makes readability worse and causes more confusion than good, as you noted it seems to work without it, and the only examples I can see of people using focus in this way are so old I'm not sure the cross browser issues referenced are even still issues.